microsoft/electionguard-ui
# Election Setup - Basic info

[//]: # (🚨 Please review the CONTRIBUTING.md in this repository. 💔Thank you!)

### Issue
Supports #87 

### Description
Replaces the setup instructions page with a basic information retrieval page that just asks for the election id.  Eventually this will ask for the election name too.  

This also starts the process of building up the object throughout the course of the individual wizard steps that will be sent to the election creation API endpoint.

### Testing
1. Log in
2. Select "Setup Election"

Expected: You should now arrive at a page called "Election Setup: Basic Info" that looks like this:

![image](https://user-images.githubusercontent.com/1769905/151215547-cee5b254-74f3-4b9a-ae1d-cf0b5bab5618.png)

4. Enter an election id
5. Hit next

Expected: you should navigate at the "Pull Guardian Keys" page
